app/assets/javascripts/recipe.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Are you sure that you want to use coffescript? This is translation language for javascript and the syntax is different. Not sure if you are familiar with it.